### PR TITLE
documentation pass 11

### PR DESCRIPTION
**Pull request by alganthe**
#### 1.1  Updated
  - information center
  - mortar doc
  - medical doc
  - maptools doc
  - huntIR doc
  - settings framework

#### 1.2 Added framework doc
 - disposables
 - explosives
 - goggles
 - hearing
 - huntIR
 - javelin
 - missileguidance
 - night vision
 - overpressure
 - overheating
 - parachute
 - reloadlaunchers
 - scopes
 - sitting

### CODE REVIEW

File: documentation/user/information-center.md [hunk 1]
```diff
@@ -39,10 +39,14 @@ Downloaded ACE3 and have no idea where to start? This page serves as a starting
 **A:** Fun fact, it isn't, you need to come from the direction of the laser, (laser is pointing to the east, you come from the west) and you drop the GBU, it will then guide itself to the target. The reasoning behind that is that the vehicle or building laser designated would obstruct the laser and the GBU would then be unable to lock on it. </br>
 
 **Q:** I take vanilla damage with ACE 3.1.1</br>
-**A:** This has been fixed on ACE3 master and will be fixed in the next release, in the meantime Basic medical doesn't have that issue. </br>
+**A:** This has been fixed in 3.2.0</br>
 
 **Q:** I'm having dll errors.</br>
-**A:** Remove ACE3 from your Arma 3 folder and repeat the installation process (don't forget to re-download).</br>
+**A:** Start the game once with the arma3 launcher, close it then open it with your usual launcher (arma3sync or PW6).</br>
```
Comment (jonpas) [inline documentation/user/information-center.md]: ...with the **Arma 3 Launcher**, close it**,** then **start the game** with your usual launcher (**ArmA3Sync** or **Play withSIx**).

File: documentation/user/information-center.md [hunk 2]
```diff
@@ -39,10 +39,14 @@ Downloaded ACE3 and have no idea where to start? This page serves as a starting
...
+The simple explanation to that is that battleye break itself and only starting it with the arma3 launcher fixes it.</br>
```
Comment (jonpas) [inline documentation/user/information-center.md]: The simple explanation **is that BattlEye breaks** itself and only starting it with the **Arma 3 Launcher** fixes it.

**Comment by alganthe** [comment]
I reviewed the whole thing, everything is normalized.
This is ready for a merge.

File: documentation/feature/medical-system.md [hunk 3]
```diff
@@ -288,5 +296,76 @@ Note that keeping the patient's vitals stable is very important while treating h
...
+
+- A unit in the revive state will be unconscious and will stay unconscious until he is either woken up or the revive timer runs out.
+- A unit in the revive state can't die from any source of damage, only the timer ending can kill it.
+- To wake up a patient the use of a PAK is required.
+- Each successful revive removes a life from the unit, once the lives run out the next time unit will take fatal damage he will not enter the revive state and will die.
+- Each successful round of CPR (filled up completion bar) increase the time left in the revive state.
+
+## 3. Guides
+### 3.1 Loadouts
+
```
Comment (gienkov) [inline documentation/feature/medical-system.md]: Can you rename to "Example loadouts" ?

File: documentation/feature/medical-system.md [hunk 4]
```diff
@@ -288,5 +296,76 @@ Note that keeping the patient's vitals stable is very important while treating h
...
+
+- Paramedic:
+  - 10-15 × Bandage (Basic)
+  - 15-20 × Bandage (Elastic)
+  - 15-20 × Packing Bandage
+  - 10-15 × Basic Field Dressing (QuikClot)
+  - 5 × Tourniquet (CAT)
+  - 2 × Saline IV (500ml)
+
```
Comment (gienkov) [inline documentation/feature/medical-system.md]: Content has been cut here or ?

Comment (alganthe) [inline documentation/feature/medical-system.md]: oh, forgot to add it back good point, give me a sec

Comment (alganthe) [inline documentation/feature/medical-system.md]: EDIT: it's there bottom of the page.....

File: documentation/feature/maptools.md [hunk 5]
```diff
@@ -16,23 +16,23 @@ This adds the possibility to draw accurate lines on the map screen.
...
 
 ### 1.3 GPS on map
-If you are equipped with a vanilla GPS it will be shown on the map. (You don't need the map tools in your inventory for this.)
+If you are equipped with a vanilla GPS it will be shown on the map. (You don't need the `Map Tools` item in your inventory for this.)
 
 ## 2. Usage
 
 ### 2.1 Using map tools
-- For this you need to have `Map Tools`.
+- To use map tools `Map Tools` in your inventory are required.
```
Comment (bux578) [inline documentation/feature/maptools.md]: according to line 19 "`Map Tools` item"

File: documentation/feature/maptools.md [hunk 6]
```diff
@@ -16,23 +16,23 @@ This adds the possibility to draw accurate lines on the map screen.
...
 - Open the map <kbd>M</kbd> (Arma 3 default key bind `Map`).
 - Press the self interaction key <kbd>Ctrl</kbd> + <kbd>⊞&nbsp;Win</kbd> (ACE3 default key bind `Self Interaction Key`).
 - Select `Map tools`.
 - Select the type of tools you want to use.
-- Note that you can drag the Roamer (map tool) around with <kdd> LMB </kbd> and rotate it with <kbd>Ctrl</kbd> + <kbd>LMB</kbd>.
+- Note that you can drag the Roamer (map tool) around with <kBd>LMB</kbd> and rotate it with <kbd>Ctrl</kbd> + <kbd>LMB</kbd>.
```
Comment (bux578) [inline documentation/feature/maptools.md]: `<kbd>`

File: documentation/feature/maptools.md [hunk 7]
```diff
@@ -16,23 +16,23 @@ This adds the possibility to draw accurate lines on the map screen.
...
 
 ### 2.2 Drawing lines
-- To draw lines `Map Tools` are not required.
+- To draw lines `Map Tools` in your inventory are required.
```
Comment (bux578) [inline documentation/feature/maptools.md]: To draw lines the `Map Tools` item needs to be equipped.

File: documentation/feature/medical-system.md [hunk 8]
```diff
@@ -1,13 +1,13 @@
 ---
 layout: wiki
 title: Medical System
-description: ACE provide users with a more realistic medical system and comes in both a basic and advanced version. Both versions have overlap but each have their own unique characteristics. 
+description: ACE3 provide users with a more realistic medical system and comes in both a basic and advanced version. Both versions have overlap but each have their own unique characteristics.
```
Comment (bux578) [inline documentation/feature/medical-system.md]: "provides"

File: documentation/feature/medical-system.md [hunk 9]
```diff
@@ -1,13 +1,13 @@
...
 group: feature
 order: 4
 category: realism
 parent: wiki
 ---
-**Disclaimer:** The documentation for the medical system being extremely long it's highly advised to use the table of contents at the top right corner of the page.
+**Disclaimer:** As the documentation for the medical system being extremely long it's highly advised to use the table of contents at the top right corner of the page.
```
Comment (bux578) [inline documentation/feature/medical-system.md]: "With the" instead of "As the"

or if you want to use "As the" then switch "being" with "is"

File: documentation/feature/medical-system.md [hunk 10]
```diff
@@ -109,6 +109,14 @@ Atropine | Serves no use in basic
 - If the revive system is in place your character will not die until the revive timer is at 0. Even if a tank shoots your ass off an epinephrine shot will bring you back up after your wounds are treated. (The timer is invisible and may vary from mission to mission, it also depends on the amount of lives remaining you have.)
 - You can't do an overdose in basic.
 
+#### 2.1.4 Revive (Basic Medical)
+For the following procedure to work revive need to be enabled.
+
+- A unit in the revive state will be unconscious and will stay unconscious until he is either woken up or the revive timer runs out.
+- A unit in the revive state can't die from any source of damage, only the timer ending can kill it.
```
Comment (bux578) [inline documentation/feature/medical-system.md]: here you're talking about the unit as "it". In the line above as "he"

File: documentation/feature/medical-system.md [hunk 11]
```diff
@@ -109,6 +109,14 @@ Atropine | Serves no use in basic
...
+- To wake up a patient treat all of his wounds, make sure he isn't in pain and then use epinephrine.
+- Each successful revive removes a life from the unit, once the lives run out the next time unit will take fatal damage he will not enter the revive state and will die.
```
Comment (bux578) [inline documentation/feature/medical-system.md]: "the next time unit will take" => "the next time the unit will take"

File: documentation/feature/medical-system.md [hunk 12]
```diff
@@ -288,5 +296,76 @@ Note that keeping the patient's vitals stable is very important while treating h
 - You don't have to take epinephrine after you take morphine, just wait until your pulse stabilizes by itself (Provided that you are in a stable condition).
 - Giving too much morphine to a patient (more than one every 10 minutes) will put him in cardiac arrest because of a blood pressure / heart rate too low.
 
-## 3. Dependencies
-`ace_interaction`, `ace_modules`, `ace_apl`
\ No newline at end of file
+#### 2.2.5 Revive (Advanced Medical)
+For the following procedure to work revive need to be enabled.
```
Comment (bux578) [inline documentation/feature/medical-system.md]: "needs".

File: documentation/feature/medical-system.md [hunk 13]
```diff
@@ -288,5 +296,76 @@ Note that keeping the patient's vitals stable is very important while treating h
...
+
+- A unit in the revive state will be unconscious and will stay unconscious until he is either woken up or the revive timer runs out.
+- A unit in the revive state can't die from any source of damage, only the timer ending can kill it.
```
Comment (bux578) [inline documentation/feature/medical-system.md]: "he" vs. "it"

File: documentation/feature/medical-system.md [hunk 14]
```diff
@@ -288,5 +296,76 @@ Note that keeping the patient's vitals stable is very important while treating h
...
+- To wake up a patient the use of a PAK is required.
+- Each successful revive removes a life from the unit, once the lives run out the next time unit will take fatal damage he will not enter the revive state and will die.
+- Each successful round of CPR (filled up completion bar) increase the time left in the revive state.
```
Comment (bux578) [inline documentation/feature/medical-system.md]: "increases"

File: documentation/feature/medical-system.md (hunk 3 above)
Comment (bux578) [inline documentation/feature/medical-system.md]: :+1:

File: documentation/feature/medical-system.md [hunk 15]
```diff
@@ -288,5 +296,76 @@ Note that keeping the patient's vitals stable is very important while treating h
...
+
+- Soldier :
+  - 3-6 × Bandage (Basic)
+  - 3-6 × Bandage (Elastic)
+  - 3-6 × Packing Bandage
+  - 3-6 × Basic Field Dressing (QuikClot)
+  - 1 × Morphine autoinjector
+  - 1 × Epinephrine autoinjector
+  - 1 × Tourniquet (CAT)
+  - **Optional**: 1 × Saline IV (500ml) - used only by qualified medic
```
Comment (bux578) [inline documentation/feature/medical-system.md]: "by a qualified"

File: documentation/feature/medical-system.md [hunk 16]
```diff
@@ -288,5 +296,76 @@ Note that keeping the patient's vitals stable is very important while treating h
...
+  - 3 × Tourniquet (CAT)
+  - 4 × Saline IV (500ml)
+
+
+- Medic:
+  - 10-15 × Bandage (Basic)
+  - 15-20 × Bandage (Elastic)
+  - 15-20 × Packing Bandage
+  - 10-15 × Basic Field Dressing (QuikClot)
+  - 12 × Atropine Autoinjector
```
Comment (bux578) [inline documentation/feature/medical-system.md]: "Atropine Autoinjector" vs "Atropine autoinjector" stick to writing "autoinjector" with a capital letter or with only lowercase

File: documentation/feature/mk6mortar.md [hunk 17]
```diff
@@ -9,57 +9,38 @@ parent: wiki
...
+For this you need a `82mm Rangetable`, `Map Tools` and a `Vector 21` are also recommended.
+
+
+- Get the distance and elevation difference between you and the target (you can use map tools).
+- Select the charge you want to use (0 = close / 1 = medium  / 2 = far).
+- Open the `82mm Rangetable`
+- Calculate the correct ELEV (elevation):
+  - Open the `Rangetable` and click on the charge you are using.
+  - Find your range under the `RANGE` column
+  - Under the `ELEV` column find the number that match your range, that's the base elevation you are going to work with.
```
Comment (bux578) [inline documentation/feature/mk6mortar.md]: that matches

File: documentation/feature/mk6mortar.md [hunk 18]
```diff
@@ -9,57 +9,38 @@ parent: wiki
...
+  - Under the `D ELEV for 100m DR` find the number that match your ELEV and compensate:
```
Comment (bux578) [inline documentation/feature/mk6mortar.md]: that matches

File: documentation/feature/mk6mortar.md [hunk 19]
```diff
@@ -9,57 +9,38 @@ parent: wiki
...
+     - Example, if you're 200m above your target multiply the number by 2 and add it to your ELEV.
```
Comment (bux578) [inline documentation/feature/mk6mortar.md]: "Example:"
maybe in bold or italic

File: documentation/framework/explosives-framework.md [hunk 20]
```diff
@@ -0,0 +1,123 @@
...
+</div>
+
+## 1. Explosives
+
+### 1.1 Setting the magazine
+
+```c++
+class CfgMagazines {
+    class CA_Magazine;
+    class banana_satchel_remote_mag: CA_Magazine {
```
Comment (bux578) [inline documentation/framework/explosives-framework.md]: a better example please. Don't treat bananas like that

File: documentation/framework/explosives-framework.md [hunk 21]
```diff
@@ -0,0 +1,123 @@
...
+    };
+};
+```
+
+### 1.3 Adding the place item
+
+```c++
+class CfgVehicles {
+    class ACE_Explosives_Place;
+    class banana_satchel_place: ACE_Explosives_Place {
```
Comment (bux578) [inline documentation/framework/explosives-framework.md]: better example pls

File: documentation/framework/explosives-framework.md [hunk 22]
```diff
@@ -0,0 +1,123 @@
...
+## 2. Mines
+
+### 2.1 Setting the magazine
+
+_Pretty much the same as Explosives except that we inherit from_ `ATMine_Range_Mag` _instead of_ `CA_Magazine`.
+
+```c++
+class CfgMagazines {
+    class ATMine_Range_Mag;
+    class BananaMine_Range_Mag: ATMine_Range_Mag {
```
Comment (bux578) [inline documentation/framework/explosives-framework.md]: better example pls

File: documentation/framework/explosives-framework.md [hunk 23]
```diff
@@ -0,0 +1,123 @@
...
+Directional mines inherit from `DirectionalBombBase`.
+
+
+## 3. Adding your own detonators
+
+```c++
+class CfgWeapons {
+    class ACE_ItemCore;  // ACE3 base item class
+    class ACE_Clacker;  // Clacker base class
+    class banana_clacker: ACE_Clacker {
```
Comment (bux578) [inline documentation/framework/explosives-framework.md]: better example pls

File: documentation/framework/huntIR-framework.md [hunk 24]
```diff
@@ -0,0 +1,27 @@
...
+```c++
+class CfgWeapons {
+  class MyRifle {
+      class MyGL: UGL_F {
+          magazines[] = {
+              // All default UGL magazines
+              "MyFirstMag",
+              "MySecondMag",
+              "MyLastMag",
+              // HUntIR magazine
```
Comment (bux578) [inline documentation/framework/huntIR-framework.md]: HuntIR

File: documentation/framework/parachute-framework.md [hunk 25]
```diff
@@ -0,0 +1,19 @@
...
+group: framework
+order: 5
+parent: wiki
+---
+
+## 1. Adding reserve parachute to existing parachute
+
+```c++
+class CfgVehicles {
+    class BananaParachute {
```
Comment (bux578) [inline documentation/framework/parachute-framework.md]: better example pls

File: documentation/framework/reloadlaunchers-framework.md [hunk 26]
```diff
@@ -0,0 +1,26 @@
...
+order: 5
+parent: wiki
+---
+
+## 1. Adding third party reload
+
+```c++
+class CfgWeapons {
+    class MyLauncher {
+        ace_reloadlaunchers_enabled = 1;  // Allow your buddies reloading for you (0-disabled, 1-enabled)
```
Comment (bux578) [inline documentation/framework/reloadlaunchers-framework.md]: Allow your "buddies" to reload your weapon

File: documentation/framework/sitting-framework.md [hunk 27]
```diff
@@ -0,0 +1,26 @@
+---
+layout: wiki
+title: Sitting Framework
+description: Explains how to set-up sitting furniture with ACE3 sitting system.
```
Comment (bux578) [inline documentation/framework/sitting-framework.md]: Explains how to set-up furniture to allow sitting with the ACE3 sitting system.

File: documentation/framework/sitting-framework.md [hunk 28]
```diff
@@ -0,0 +1,26 @@
...
+group: framework
+order: 5
+parent: wiki
+---
+
+## 1. Adding sitting support to a chair
+
+<div class="panel callout">
+    <h5>Note:</h5>
+    <p>Unfinished! What you see below will not work in ACE 3.2.1</p>
```
Comment (bux578) [inline documentation/framework/sitting-framework.md]: ... then throw this file out and put it in a different PR that will be merged after 3.3.0

File: documentation/user/how-to-report-an-issue.md [hunk 29]
```diff
@@ -24,27 +24,29 @@ Indicate if the issue appears on stable or development version. In case it is th
 Head over to the <a href="{{ site.githubUrl }}/issues" target="_blank">ACE3 GitHub issue tracker</a> and press the <a href="{{ site.githubUrl }}/issues/new" target="_blank">"New issue"</a> button in the top right corner. Add a descriptive title and copy the following issue template in to the text area:
 
 ```
+*PRO-TIP: The english language allows individuals to correct each other's mistakes in a respectful manner.*
```
Comment (bux578) [inline documentation/user/how-to-report-an-issue.md]: pls remove this

File: documentation/user/how-to-report-an-issue.md [hunk 30]
```diff
@@ -24,27 +24,29 @@ Indicate if the issue appears on stable or development version. In case it is th
...
 **Description:**
-*Add a detailed description of the error. This makes it easier for us to fix the issue.*
+* Add a detailed description of the error. This makes it easier for us to fix the issue.*
 
 **Steps to reproduce:**
 * *Add the steps needed to reproduce the issue.*
 
 **Where did the issue occur?**
-*A possible answer might be "Multiplayer", "Singleplayer"*
+* A possible answer might be "Multiplayer" or "Singleplayer" or "Editor"*
```
Comment (bux578) [inline documentation/user/how-to-report-an-issue.md]: "Multiplayer", "Editor" or "Singleplayer"*

File: documentation/user/information-center.md [hunk 31]
```diff
@@ -39,15 +39,19 @@ Downloaded ACE3 and have no idea where to start? This page serves as a starting
 **A:** Fun fact, it isn't, you need to come from the direction of the laser, (laser is pointing to the east, you come from the west) and you drop the GBU, it will then guide itself to the target. The reasoning behind that is that the vehicle or building laser designated would obstruct the laser and the GBU would then be unable to lock on it. </br>
 
 **Q:** I take vanilla damage with ACE 3.1.1</br>
-**A:** This has been fixed on ACE3 master and will be fixed in the next release, in the meantime Basic medical doesn't have that issue. </br>
+**A:** This has been fixed in 3.2.0</br>
```
Comment (bux578) [inline documentation/user/information-center.md]: If it's fixed, remove the whole thing :)

File: documentation/user/information-center.md [hunk 32]
```diff
@@ -39,15 +39,19 @@ Downloaded ACE3 and have no idea where to start? This page serves as a starting
...
 
-**Q:** I'm having dll errors.</br>
-**A:** Remove ACE3 from your Arma 3 folder and repeat the installation process (don't forget to re-download).</br>
+**Q:** I am having dll errors.</br>
+**A:** Start the game once with the Arma 3 Launcher, close it, then start the game with your usual launcher (ArmA3Sync or Play withSIx).</br>
```
Comment (bux578) [inline documentation/user/information-center.md]: `(ArmA3Sync,Play withSix, etc &hellip;)`  (note the second lowercase "i" in "withSix")

File: documentation/user/information-center.md [hunk 33]
```diff
@@ -39,15 +39,19 @@ Downloaded ACE3 and have no idea where to start? This page serves as a starting
...
+The simple explanation is that BattlEye breaks itself and only starting it with the Arma 3 Launcher fixes it.</br>
```
Comment (bux578) [inline documentation/user/information-center.md]: breaks itself? pls find a better explanation

File: documentation/user/information-center.md [hunk 34]
```diff
@@ -39,15 +39,19 @@ Downloaded ACE3 and have no idea where to start? This page serves as a starting
...
+
+**Q:** ACE3 fonts is outdated.</br>
+**A:** This happens because you have ACE3 development environment set-up, which means file patching is enabled. Use `-noFilePatching` startup parameter (in the upcoming Arma 3 update this will not be needed as file patching will be disabled by default).
```
Comment (bux578) [inline documentation/user/information-center.md]: can be removed ... 1.50 disabled file patching as stated here

File: documentation/user/information-center.md [hunk 35]
```diff
@@ -39,15 +39,19 @@ Downloaded ACE3 and have no idea where to start? This page serves as a starting
...
 
 ### 1.3 Compatibility
 
-**Q:**(mod) doesn't have some ACE3 features.</br>
-**A:**ACE3 isn't and can't be responsible for compatibility with every (mod), due it's size other (mod) authors are strongly encouraged to provide that from their side. Compatibility PBOs currently in ACE3 are there to kick-start and provide examples for (mod) authors.</br>
+**Q:** (mod) doesn't have some ACE3 features.</br>
+**A:** ACE3 isn't and can't be responsible for compatibility with every (mod), due it's size other (mod) authors are strongly encouraged to provide that from their side. Compatibility PBOs currently in ACE3 are there to kick-start and provide examples for (mod) authors.</br>
```
Comment (bux578) [inline documentation/user/information-center.md]: PBO's

File: documentation/framework/sitting-framework.md (hunk 28 above)
Comment (jonpas) [inline documentation/framework/sitting-framework.md]: My bad, I hoped to finish this way ago, I didn't.

File: documentation/feature/medical-system.md (hunk 16 above)
Comment (jonpas) [inline documentation/feature/medical-system.md]: I went for lower-case because it's named like that in the actual game/stringtable.
